<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Eggs3</class>
 <widget class="QWidget" name="Eggs3">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gen 3 Eggs</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBoxProfile">
     <property name="title">
      <string>Profile</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_13">
      <item row="1" column="3">
       <widget class="QLabel" name="labelProfileSIDValue">
        <property name="text">
         <string notr="true">54321</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="labelProfileTIDValue">
        <property name="text">
         <string notr="true">12345</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4" rowspan="2">
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="6">
       <widget class="QLabel" name="labelProfileGameValue">
        <property name="text">
         <string>Emerald</string>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="QLabel" name="labelProfileGame">
        <property name="text">
         <string>Game</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="labelProfileSID">
        <property name="text">
         <string>SID</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="pushButtonProfileManager">
        <property name="text">
         <string>Manager</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="labelProfile">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Profile</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="comboBoxProfiles"/>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="labelProfileTID">
        <property name="text">
         <string>TID</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QTabWidget" name="tabEggSelection">
     <widget class="QWidget" name="tabEmerald">
      <attribute name="title">
       <string>Emerald</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_17">
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBoxEmeraldRNGInfo">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>1</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>RNG Info</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="4" column="0" colspan="3">
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="labelEmeraldRedraws">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Redraws</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="TextBox" name="textBoxMinRedraws">
              <property name="text">
               <string notr="true">0</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="TextBox" name="textBoxMaxRedraws">
              <property name="text">
               <string notr="true">5</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="3" column="2">
           <widget class="TextBox" name="textBoxCalibration">
            <property name="text">
             <string notr="true">18</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1" colspan="2">
           <widget class="ComboBox" name="comboBoxEmeraldMethod">
            <item>
             <property name="text">
              <string>PID</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Normal</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Split</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Alternate</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="labelEmeraldInitialAdvances">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Initial Advances</string>
            </property>
           </widget>
          </item>
          <item row="6" column="0" colspan="3">
           <widget class="QPushButton" name="pushButtonEmeraldGenerate">
            <property name="text">
             <string>Generate</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="labelEmeraldMethod">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Method</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1" colspan="2">
           <widget class="TextBox" name="textBoxEmeraldInitialAdvances">
            <property name="text">
             <string notr="true">1000</string>
            </property>
           </widget>
          </item>
          <item row="5" column="0" colspan="2">
           <widget class="QLabel" name="labelEmeraldCompatibility">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Compatibility</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="labelEmeraldMaxAdvances">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Max Advances</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="labelEmeraldCalibration">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Calibration</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1" colspan="2">
           <widget class="TextBox" name="textBoxEmeraldMaxAdvances">
            <property name="text">
             <string notr="true">5000</string>
            </property>
           </widget>
          </item>
          <item row="5" column="2">
           <widget class="ComboBox" name="comboBoxEmeraldCompatibility">
            <item>
             <property name="text">
              <string>The two don't seem to like each other</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>The two seem to get along</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>The two seem to get along very well</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QGroupBox" name="groupBoxEmeraldSettings">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>2</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Settings</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="0" column="0">
           <widget class="EggSettings" name="eggSettingsEmerald" native="true"/>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QGroupBox" name="groupBoxEmeraldFilters">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>2</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Filters</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="0">
           <widget class="Filter" name="filterEmerald" native="true"/>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0" colspan="3">
        <widget class="TableView" name="tableViewEmerald">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>1</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="contextMenuPolicy">
          <enum>Qt::CustomContextMenu</enum>
         </property>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabRSFRLG">
      <attribute name="title">
       <string>RS/FRLG</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_15">
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBoxRNGInfoRSFRLG">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>1</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>RNG Info</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_11">
          <item row="0" column="0" colspan="2">
           <layout class="QGridLayout" name="gridLayout_6">
            <item row="0" column="0">
             <widget class="QLabel" name="labelSeedRSFRLG">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Seed (Held / Pickup)</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="TextBox" name="textBoxSeedHeldRSFRLG">
              <property name="text">
               <string notr="true"/>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="TextBox" name="textBoxSeedPickupRSFRLG">
              <property name="text">
               <string notr="true"/>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="labelAdvanceHeldRSFRLG">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Held Advances</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="TextBox" name="textBoxInitialAdvancesHeldRSFRLG">
              <property name="text">
               <string notr="true">1000</string>
              </property>
             </widget>
            </item>
            <item row="1" column="2">
             <widget class="TextBox" name="textBoxMaxAdvancesHeldRSFRLG">
              <property name="text">
               <string notr="true">5000</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="labelPickupAdvanceRSFRLG">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Pickup Advances</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="TextBox" name="textBoxInitialAdvancesPickupRSFRLG">
              <property name="text">
               <string notr="true">1000</string>
              </property>
             </widget>
            </item>
            <item row="2" column="2">
             <widget class="TextBox" name="textBoxMaxAdvancesPickupRSFRLG">
              <property name="text">
               <string notr="true">5000</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="labelRSMethod">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Method</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="ComboBox" name="comboBoxMethodRSFRLG">
            <item>
             <property name="text">
              <string>Normal</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Split</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Alternate</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Mixed</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="labelCompatibilityRSFRLG">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Compatibility</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="ComboBox" name="comboBoxCompatibilityRSFRLG">
            <item>
             <property name="text">
              <string>The two don't seem to like each other</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>The two seem to get along</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>The two seem to get along very well</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="3" column="0" colspan="2">
           <widget class="QPushButton" name="pushButtonGenerateRSFRLG">
            <property name="text">
             <string>Generate</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QGroupBox" name="groupBoxSettingsRSFRLG">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>2</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Settings</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_7">
          <item row="0" column="0">
           <widget class="EggSettings" name="eggSettingsRSFRLG" native="true"/>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QGroupBox" name="groupBoxFiltersRSFRLG">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>2</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Filters</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0" rowspan="3" colspan="2">
           <widget class="Filter" name="filterRSFRLG" native="true"/>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0" colspan="3">
        <widget class="TableView" name="tableViewRSFRLG">
         <property name="contextMenuPolicy">
          <enum>Qt::CustomContextMenu</enum>
         </property>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TextBox</class>
   <extends>QLineEdit</extends>
   <header>Forms/Controls/TextBox.hpp</header>
  </customwidget>
  <customwidget>
   <class>ComboBox</class>
   <extends>QComboBox</extends>
   <header>Forms/Controls/ComboBox.hpp</header>
  </customwidget>
  <customwidget>
   <class>TableView</class>
   <extends>QTableView</extends>
   <header>Forms/Controls/TableView.hpp</header>
  </customwidget>
  <customwidget>
   <class>Filter</class>
   <extends>QWidget</extends>
   <header>Forms/Controls/Filter.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>EggSettings</class>
   <extends>QWidget</extends>
   <header>Forms/Controls/EggSettings.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>comboBoxProfiles</tabstop>
  <tabstop>pushButtonProfileManager</tabstop>
  <tabstop>tabEggSelection</tabstop>
  <tabstop>textBoxEmeraldInitialAdvances</tabstop>
  <tabstop>textBoxEmeraldMaxAdvances</tabstop>
  <tabstop>comboBoxEmeraldMethod</tabstop>
  <tabstop>textBoxCalibration</tabstop>
  <tabstop>textBoxMinRedraws</tabstop>
  <tabstop>textBoxMaxRedraws</tabstop>
  <tabstop>comboBoxEmeraldCompatibility</tabstop>
  <tabstop>pushButtonEmeraldGenerate</tabstop>
  <tabstop>tableViewEmerald</tabstop>
  <tabstop>textBoxSeedHeldRSFRLG</tabstop>
  <tabstop>textBoxSeedPickupRSFRLG</tabstop>
  <tabstop>textBoxInitialAdvancesHeldRSFRLG</tabstop>
  <tabstop>textBoxMaxAdvancesHeldRSFRLG</tabstop>
  <tabstop>textBoxInitialAdvancesPickupRSFRLG</tabstop>
  <tabstop>textBoxMaxAdvancesPickupRSFRLG</tabstop>
  <tabstop>comboBoxMethodRSFRLG</tabstop>
  <tabstop>comboBoxCompatibilityRSFRLG</tabstop>
  <tabstop>pushButtonGenerateRSFRLG</tabstop>
  <tabstop>tableViewRSFRLG</tabstop>
 </tabstops>
 <resources/>
 <connections/>
 <slots>
  <slot>emeraldAnyAbility()</slot>
 </slots>
</ui>
